module/batch-action/src/Resources/config/services.yml

```diff
@@ -4,6 +4,41 @@ services:
         autoconfigure: false
         public: false
 
+    _instanceof:
+        Ergonode\BatchAction\Infrastructure\Provider\BatchActionProcessorInterface:
+            tags: ['batch_action.processor_provider.interface']
+
+    Ergonode\BatchAction\Application\Controller\Api\CreateBatchAction:
```

Comment:
I think we should create services ids with underscored values, not the FQCN's. What do you think?

Reply:
i prefer full class names its more readable for me, and whole application using this notation at this moment.
Not sure its topic for this task.

Reply:
It's not really the question of readability but rather being appropriate. "id" approach is more extendability focused - services are created with a potentially parametrized class so it can be easily overridden.  It's also allowing the possibility of not claiming of being one(via FQCN service id) but being a different class which would just be weird.

But agreed. We can go with it for different discussion